src/renderer/components/cluster-manager/cluster-status.tsx

```diff
@@ -55,47 +52,48 @@ export class ClusterStatus extends React.Component<Props> {
   }
 
   reconnect = async () => {
-    this.authOutput = [{ data: "Reconnecting..." }];
     this.isReconnecting = true;
     await this.refreshCluster();
     this.isReconnecting = false;
   }
 
   render() {
     const { authOutput, cluster, hasErrors } = this;
-    const isDisconnected = !!cluster.disconnected;
     const failureReason = cluster.failureReason;
-    const isError = hasErrors || isDisconnected;
-    return (
-      <div className={cssNames("ClusterStatus flex column gaps box center", this.props.className)}>
-        {isError && (
-          <Icon
-            material="cloud_off"
-            className={cssNames({ error: hasErrors })}
-          />
-        )}
-        <h2>
-          {cluster.preferences.clusterName}
-        </h2>
-        {!isDisconnected && (
+    if (!hasErrors || this.isReconnecting) {
+      return (
+        <div className="ClusterStatus flex column align-center justify-center">
```

Comment:
Maybe better to extract contents to some `renderContent()`? Don't like when top wrapper component is repeated..

Reply:
Moved content into `renderContent()`.